handlers/whatsapp/whatsapp.go

```diff
@@ -692,13 +698,22 @@ func sendWhatsAppMsg(msg courier.Msg, sendPath *url.URL, token string, payload i
 	return "", externalID, []*courier.ChannelLog{log}, err
 }
 
-func buildWhatsAppRequestHeader(token string) http.Header {
+func addAuthorizationHeader(header http.Header, channel courier.Channel, token string) http.Header {
```

Comment:
Probably better named something like `buildAuthorizationHeader` or `getAuthorizationHeader` since it isn't actually modifying anything which is implied with the `add..` naming.

Reply:
yeah, it makes sense. done!